src/Core/KeyManagement/UserKey/IRotateUserKeyCommand.cs

```diff
@@ -1,9 +1,9 @@
-﻿using Bit.Core.Auth.Models.Data;
-using Bit.Core.Entities;
+﻿using Bit.Core.Entities;
+using Bit.Core.KeyManagement.Models.Data;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.Data.SqlClient;
 
-namespace Bit.Core.Auth.UserFeatures.UserKey;
+namespace Bit.Core.KeyManagement.UserFeatures.UserKey;
```

Comment:
⛏️ Namespace doesn't correspond to file location, due to `UserFeatures` not existing under `KeyManagement`